libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java

```diff
@@ -194,30 +194,45 @@ private String operationDescription(String methodName) {
         return methodName.substring(methodName.indexOf('$'));
     }
 
-    public void checkNetworkAccess(Class<?> callerClass, int actions) {
+    public void checkInboundNetworkAccess(Class<?> callerClass) {
+        checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
+    }
+
+    public void checkOutboundNetworkAccess(Class<?> callerClass) {
+        checkEntitlementPresent(callerClass, OutboundNetworkEntitlement.class);
+    }
+
+    public void checkAllNetworkAccess(Class<?> callerClass) {
```

Comment:
I was expecting this to look like the following; is there a reason it needs to be more complex? If so, perhaps it could benefit from a clarifying comment.

```
checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
checkEntitlementPresent(callerClass, InboundNetworkEntitlement.class);
```